classes/02class/exercises/c02-network02/carloshz4/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|

```

Comment:
I think the AWS Reserved is the other way around -- this is the number of non-reserved.

Reply:
Fair enough, I thought that'd be always the same value. As I understand it'll be 5 IP's (including network IP, GW, .2 and .3 used by AWS, and the broadcast one).  So, would you say it's 5 all the time?